docs/create/manual.md

```diff
@@ -1,18 +1,43 @@
-# Create a Truss from a serialized model
-
-Command line: truss init, makes the structure of a Truss
-
-* Start with structure of model file and fill it in
-* Have to understand internals of Truss and Model
-    * Model file with specific functions:
-        * Load, predict, pre-process, post-process, constructor
-    * Model gets access to certain things
-        * Config, directory `data`, secrets
-    * Access a model binary
-        * Generated automatically by mk_truss
-        * Like pickling a model, tensorflow save-model, protobuff, pytorch has a mechanism...
-        * Serialized model
-        * De-serialize it in model class
-        * Download serialized model from internet
-    * Function called load, called once, before any prediction
-    * Constructor vs load means you have flexibility on when you download & deserialize, or on-demand on first prediction (like a cold start)
+# Manually
+
+Creating a Truss manually, from a serialized model, works with any model-building framework, including from-scratch bespoke models.
+
+To get started, initialize the Truss with the following command:
+
+```
+truss init
+```
+
+### Truss structure
+
+To build a Truss manually, you have to understand the package in much more detail than using it with a supported framework. Fortunately, that's why we wrote this doc.
+
+To familiarize yourself with the structure of Truss, review the [structure reference](../reference/structure.md). A Truss only has three files that you need to interact with, and this tutorial is an opinionated guide to working through them.
```

Comment:
I wonder if -- truss only has three files -- can be confusing, because that's not always the case. Perhaps we can qualify with: `For this tutorial`.